Modules/Devices/test/Devices.Domain.Tests/Tiers/TierTests.cs

```diff
@@ -30,6 +30,17 @@ public void Basic_Tier_cannot_be_deleted()
         error.Should().BeEquivalentTo(DomainErrors.CannotDeleteBasicTier());
     }
 
+    [Fact]
+    public void Queued_for_deletion_tier_cannot_be_deleted()
+    {
+        // Arrange & Act
+        var error = Tier.QUEUED_FOR_DELETION.CanBeDeleted(clientsCount: 0, identitiesCount: 0);
+
+        // Assert
+        error.Should().NotBeNull();
+        error.Should().BeEquivalentTo(DomainErrors.CannotDeleteQueuedForDeletionTier());
```

Comment:
Test for the hardcoded error code. The problem with the current assertion is that we may accidentally change the error code. And since the error code is kind of our public API, we should be fully aware of it. So we WANT the test to fail.
Can you do the same change for the other tests in this file please?